src/python/pants/backend/graph_info/tasks/dependees.py

```diff
@@ -55,10 +55,10 @@ def console_output(self, _):
       yield json.dumps(deps, indent=4, separators=(',', ': '), sort_keys=True)
     else:
       if self._closed:
-        for root in roots:
+        for root in sorted(roots):
```

Comment:
Since this fixes an error, I would strongly appreciate a comment explaining why the sorting is necessary, especially since I believe most contributors to this codebase are less familiar with python 3 idiosyncracies in general than 2.

Reply:
Good catch! I completely agree. Adding.